TeamCode/src/main/java/org/firstinspires/ftc/teamcode/bots/FrenzyBot.java

```diff
@@ -86,8 +92,53 @@
     private static double INTAKE_SPEED = -0.3;
     private static double INTAKE_SPEED_REVERSE = 0.3;
 
+    private boolean firstDuckDropped = false;
     private boolean intakeRunning = false;
 
+    private void applyFrenzyConfig() {
+        // Dropper
+        DROPPER_SERVO_POS_DROP = frenzyConfig.getDropperPositionDrop();
```

Comment:
Should we check the incoming numbers for validity? What if the file is corrupt or empty? What is the best fail-safety mechanism, e.g. check for 0 values and use hard-coded numbers instead?

Reply:
as we move forward, the java code values will progressively get more and more outdated.
If we fall back on the hard-coded numbers, the robot's behavior will be VERY unpredictable.

We could put a process in place where we periodically update the hard-coded values from the JSON file. But I'd much rather have robot fail **completely** if the JSON is missing.